### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by yaosiqian**
### Please confirm this pull request meets the following requirements:

- [x] I followed the contributing guidelines: <https://github.com/github/explore/blob/main/CONTRIBUTING.md>.
- [x] I have no affiliation with the project I am suggesting (as a maintainer, creator, contractor, or employee).

### Which change are you proposing?

  - [x] Suggesting edits to an existing topic or collection
  - [ ] Curating a new topic or collection
  - [ ] Something that does not neatly fit into the binary options above

---

### Editing an existing topic or collection

I'm suggesting these edits to an existing topic or collection:
- [ ] Image (and my file is `*.png`, square, dimensions 288x288, size <= 75 kB)
- [x] Content (and my changes are in `index.md`)

> Please replace this line with an explanation of why you think these changes should be made.

### Curating a new topic or collection

- [ ] I've formatted my changes as a new folder directory, named for the topic or collection as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]` or `https://github.com/collections/[NAME]`)
- [ ] My folder contains a `*.png` image (if applicable) and `index.md`
- [ ] All required fields in my `index.md` conform to the Style Guide and API docs: <https://github.com/github/explore/tree/main/docs>

> Please replace this line with an explanation of why you think this topic or collection should be curated.

### Something that does not neatly fit into the binary options above

- [ ] My suggested edits are not about an existing topic or collection, or at least not a single one
- [ ] My suggested edits are not about curating a new topic or collection, or at least not a single one
- [ ] My suggested edits conform to the Style Guide and API docs: https://github.com/github/explore/tree/main/docs

> Please replace this line with an explanation of your proposed changes.

---

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

### CODE REVIEW

File: collections/hacking-minecraft/index.md [hunk 1]
```diff
@@ -1,34 +1,43 @@
...
- - PaperMC/Paper
  - CaffeineMC/sodium-fabric
- - FabricMC/fabric
- - lambda-client/lambda/
+ - lambda-client/lambda
  - nerdsinspace/nocom-explanation
+ - CCBlueX/LiquidBounce
 display_name: Hacking Minecraft
-created_by: leereilly
+created_by: leereilly, YaoSiQian
```
Comment (MikeMcQuaid) [inline collections/hacking-minecraft/index.md:39]: You're adding to this but you didn't create it.

```suggestion
created_by: leereilly
```

File: collections/hacking-minecraft/index.md [hunk 2]
```diff
@@ -1,34 +1,43 @@
...
 image: hacking-minecraft.png
 ---
 Minecraft is a game about building blocks, but it doesn’t end there. Take Minecraft further with some of the projects below, or dive into the code mines and hammer your own!
+With these projects' help, you may have a better experience!
```
Comment (MikeMcQuaid) [inline collections/hacking-minecraft/index.md:43]: This addition is unnecessary.
```suggestion
```

File: collections/hacking-minecraft/index.md [hunk 3]
```diff
@@ -1,34 +1,43 @@
 ---
 items:
  - docker/dockercraft
+ - Bukkit/Bukkit
```
Comment (MikeMcQuaid) [inline collections/hacking-minecraft/index.md:4]: Please restore the ordering from before so we can see what you've added.

**Review by MikeMcQuaid — CHANGES_REQUESTED** [review]
Thanks for the PR! Need a few changes before we can merge this I'm afraid.

**Comment by ponratm** [comment]
@tommorris[https://goo.gl/maps/BGS68wQbhtxjv1g68](https://github.com/explore) 
 https://goo.gl/maps/BGS68wQbhtxjv1g68

**Comment by ponratm** [comment]
IMG_20220617_044041
Screenshot_2022-06-17-11-27-27-050_com miui home
แตกไฟเอง
IMG_20220616_120132~2

File: collections/hacking-minecraft/index.md (hunk 1 above)
Comment (Athayde3586) [reply collections/hacking-minecraft/index.md:39]: Hello hacker

**Comment by K7ilan72** [comment]
Good

**Pull request closed without merging**